File: Lógica de Programação/Aula03/exercicio02.py
#Faça um código que receba as 3 notas de um aluno e
# verifique se ele está aprovado ou reprovado.
# Considere que a média para aprovação é 7,0.

#Variáveis para notas.
nota1 = float(input("Digite a primeira nota:"))
nota2 = float(input("Digite a segunda nota:"))
nota3 = float(input("Digite a segunda nota:"))

#variável para calcular a média
media = (nota1 + nota2 + nota3) / 3

#Decição para saber se o aluno foi aprovado ou reprovado e saida do resultado
if media >= 7:
    print("Aluno aprovado")
else:
    print("Aluno reprovado")

File: Java/Turma_D/src/array/exercicio02.java
package array;

import java.util.Scanner;

public class exercicio02 {

	public static void main(String[] args) {
		double[] nota = new double[5];
		double soma = 0;
		double media = 0;
		
		Scanner input = new Scanner(System.in);
		
		for(int i=0; i<nota.length; i++) {
			System.out.print("Nota do aluno: ");
			nota[i] = input.nextDouble();	 
		}
		
		for(int x=0; x<nota.length; x++) {
			soma += nota[x];
				
		}
		media = soma / nota.length;
		
		System.out.println(soma);
	}
}
